lib/adyen/rest/request.rb

```diff
@@ -29,17 +29,19 @@ module REST
     # @see Adyen::REST::Client
     # @see Adyen::REST::Response
     class Request
-      attr_reader :prefix, :form_data, :required_attributes
+      attr_reader :prefix, :form_data, :required_attributes, :resource
       attr_accessor :response_class, :response_options
 
       def initialize(action, attributes, options = {})
-        @prefix = options[:prefix]
-        @form_data = generate_form_data(action, attributes)
+        raise Adyen::REST::RequiredParameterMissing, "actions is required!" if action.nil?
```

Comment:
Sure that would be complicated to do that, but if we know something is mandatory, I'd prefer throwing an explicit message about it. 

Otherwise I could use `action:` to let ruby throw an `ArgumentError`. But this will break backward compatibility if some one use directly the request class (`Adyen::REST::Request.new`) outside of this lib.